tests/model/schema.js

```diff
@@ -1101,7 +1123,13 @@ describe( 'Schema', () => {
 				}
 			} );
 
-			setData( model, '<p>foo<img />bar</p>' );
+			// Parse data string to model.
+			const parsedResult = setData._parse( '[<p>foo<img />bar</p>]', model.schema, { context: [ root.name ] } );
+
+			// Set parsed model data to prevent selection post-fixer from running.
+			model.change( writer => {
+				writer.insert( parsedResult.model, root );
```

Comment:
I don't really know how to do that  - AFAICS it's easiest to just use `writer.insert( node, root)`...

Reply:
The method we're testing  there `schema.getValidRanges()` doesn't need to work on on a piece of a tree which is in the model. At least – I think that doesn't need that. It should be able to work on a detached tree. 

OTOH, we can leave  it with the `insert()` and test it just like it will be used most of the time (in an attached tree).